components/google-built-opentelemetry-collector/exporter/googleservicecontrolexporter/factory.go

```diff
@@ -132,10 +133,15 @@ func createMetricsExporter(ctx context.Context, settings exporter.Settings, cfg
 }
 
 func createClient(ctx context.Context, oCfg *Config, settings exporter.Settings) (*ServiceControlClient, error) {
-	opts := []grpc.DialOption{}
+	userAgent := fmt.Sprintf("%s/%s (%s/%s)",
+		settings.BuildInfo.Description, settings.BuildInfo.Version, runtime.GOOS, runtime.GOARCH)
+	opts := []grpc.DialOption{
+		grpc.WithUserAgent(userAgent),
+	}
+	useRawServiceControlClient := strings.TrimSpace(strings.ToLower(oCfg.UseRawServiceControlClient)) == "true"
```

Comment:
Nit: why is `oCfg.UseRawServiceControlClient` a string and not a bool? If it does need to accept values other than true or false, can we check in `Validate()` that it conforms to the rules here (is lower case and has no surrounding whitespace)? It'd be better if we didn't have to trim and normalize the string at the point of check.